loggo/loggo.py

```diff
@@ -19,10 +19,10 @@
     graypy = None
 
 # Strings to be formatted for pre function, post function and error during function
-FORMS = dict(pre='*Called {call_signature}',
-             post='*Returned from {call_signature} with {return_type} {return_value}',
-             noreturn='*Returned None from {call_signature}',
-             error='*Errored during {call_signature} with {error_type} "{error_string}"')
+FORMS = dict(called='*Called {call_signature}',
+             returned='*Returned from {call_signature} with {return_type} {return_value}',
+             returned_none='*Returned None from {call_signature}',
```

Comment:
To simplify things, what if we didn't have this at all, but just `returned`... Would life be worse?

Reply:
it just looks too shit, sorry. i want this.